app/account_linking.py

```diff
@@ -38,6 +51,38 @@ class LinkResult:
     strategy: str
 
 
+def send_double_subscription_if_needed(partner_user: PartnerUser):
+    sub = partner_user.user.get_active_subscription()
+    if isinstance(sub, CoinbaseSubscription):
+        channel = "Coinbase"
+    elif isinstance(sub, AppleSubscription):
+        channel = "Apple"
+    elif isinstance(sub, Subscription):
+        channel = "Paddle"
+    else:
+        return
+    send_email_at_most_times(
```

Comment:
do you know why do we use `send_email_at_most_times` and not just `send_email` here?

Reply:
We said that we only wanted to send it once even if the user re-linked. Do you prefer to send a mail at each update? Keep in mind that a renewal in the partner side will also trigger this code.

Reply:
ah ok, let's send it only once then.

Reply:
:+1: